docs/source/cli.rst

```diff
@@ -8,139 +8,12 @@ serve MLflow Python Function and scikit-learn models, and serve models on
 `Microsoft Azure Machine Learning <https://azure.microsoft.com/en-us/overview/machine-learning/>`_ and
 `Amazon SageMaker <https://aws.amazon.com/sagemaker/>`_.
 
-.. code-block:: bash
-
-    $ mlflow --help
-    Usage: mlflow [OPTIONS] COMMAND [ARGS]...
-
-    Options:
-      --version  Show the version and exit.
-      --help     Show this message and exit.
-
-    Commands:
-      azureml      Serve models on Azure Machine Learning.
-      download     Download the artifact at the specified DBFS or S3 URI. 
-      experiments  Manage experiments.
-      pyfunc       Serve Python models locally.
-      run          Run an MLflow project from the given URI.
-      sagemaker    Serve models on Amazon SageMaker.
-      sklearn      Serve scikit-learn models.
-      ui           Run the MLflow tracking UI.
-
-
 Each individual command has a detailed help screen accessible via ``mlflow command_name --help``.
 
 .. contents:: Table of Contents
   :local:
   :depth: 2
 
-
-Azure Machine Learning Models
------------------------------
-
-Subcommands to serve models on Azure Machine Learning.
-
-
-Download
---------
-
-Download the artifact at the specified DBFS or S3 URI into the specified
-local output path, or the current directory if no output path is
-specified.
-
-
-Experiments
------------
-
-Subcommands to manage experiments.
-
-
-.. contents:: In this section:
-  :local:
-  :depth: 1
-
-Create
-~~~~~~
-
-Create an experiment. The command has required argument for experiment name.
-Additionally, you can provide an artifact location  using ``-l`` or ``--artifact-location``
-option. If not provided, backend store will pick default location. Backend store will generate a
-unique ID for each experiment.
-
-All artifacts generated by runs related to this experiment will be stored under artifact location,
-organized under specific run_uuid sub-directories.
-
-Implementation of experiment and metadata store is dependent on backend storage. ``FileStore``
-creates a folder for each experiment ID and stores metadata in ``meta.yaml``. Runs are stored as
-subfolders.
-
-
-List
-~~~~
-
-Lists all experiments managed by backend store. Command takes an optional ``--view`` or ``-v``
-argument. Valid arguments are ``active_only`` (default), ``deleted_only``, or ``all``.
-
-
-Delete
-~~~~~~
-
-Mark an active experiment for deletion. This also applies to experiment's metadata, runs and
-associated data, and artifacts if they are store in default location. Use ``list`` command to view
-artifact location. Command takes a required argument for experiment ID. Command will thrown
-an error if experiment is not found or already marked for deletion.
-
-Experiments marked for deletion can be restored using ``restore`` command, unless they are
```

Comment:
We should avoid losing some of these docs which are not in the CLI today. Specifically, for delete we currently refer to `restore_experiment` in the CLI docs (which doesn't exist) and don't mention the implementation of deletion stores. Mind adding these to the CLI?

Reply:
yeah the plan was to add stuff from the old version of `cli.rst` into the CLI docstrings if we were okay with this autogen route (didn't want to do the work prematurely :P)

Reply:
This looks good so far!